crates/next-dev/benches/bundlers/mod.rs

```diff
@@ -29,6 +29,10 @@ pub trait Bundler {
     fn has_server_rendered_html(&self) -> bool {
         false
     }
+    /// There is a hydration done event emitted by client side JavaScript
+    fn has_interactivity(&self) -> bool {
+        true
```

Comment:
You could change `has_interactivity` and `has_server_rendered_html` to return an enum.

```rust
enum InteractivityMode {
    None,
    HydrationDone,
}

impl InteractivityMode {
    pub fn is_interactive(&self) -> bool {
        match self { ... }
    }
}

enum RenderMode {
    Client,
    Server,
}

// etc.
```

This will make the following code clearer to understand, without having to know the order of arguments to `Turbopack::new`.

Reply:
I'll do